Libraries/Components/WebView/WebView.ios.js

```diff
@@ -334,6 +334,21 @@ class WebView extends React.Component {
      * to tap them before they start playing. The default value is `true`.
      */
     mediaPlaybackRequiresUserAction: PropTypes.bool,
+
+    /**
+     * Rules that will be used to block the webview navigation
+     */
+    navigationBlockingPolicies: PropTypes.arrayOf(PropTypes.shape({
+      currentURL: PropTypes.string,
+      navigationType: PropTypes.string,
+      url: PropTypes.string,
```

Comment:
Why is there `navigationType` only on iOS? What are the possible values? Add docs please.